doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -492,11 +509,34 @@ dpu_data_plane_state: up  refers to configuration downloaded, the pipeline stage
 ```
 
 3. Each DPU has to store the health data in its local DB and should provide it to the switch.
-* When the "show system-health ..." CLI is executed on the switch, the "UserDefinedChecker" class will collect the data and feed it to the CLI. It is up to the platform on how this is done.  However, for faster access store it in the switch ChassisStateDB.
+* When the "show system-health ..." CLI is executed on the switch. For faster access store it in the switch ChassisStateDB.
 * The DPU is a complex hardware, to facilitate debug, a consistent way of storing and accessing the health record of the DPUs is critical in a multi vendor scenario even though it is a platform specific implementation.
-* Both switch and the DPUs will follow to the [SONiC system health monitor HLD](https://github.com/sonic-net/SONiC/blob/ce313db92a694e007a6c5332ce3267ac158290f6/doc/system_health_monitoring/system-health-HLD.md)
+* Both switch and the DPUs will follow the [SONiC system health monitor HLD](https://github.com/sonic-net/SONiC/blob/ce313db92a694e007a6c5332ce3267ac158290f6/doc/system_health_monitoring/system-health-HLD.md)
 * Refer to section 3.4.5 for "show system-health .." CLIs
 
+#### Schema for HEALTH_INFO of DPUs on switch ChassisStateDB
```

Comment:
Currently, in SONiC, system health information is stored in the STATE_DB. It is proposed to have this information in the CHASSIS_STATE_DB. Does this mean that the information will be duplicated in both databases?

Reply:
The host  system health information is stored in the STATE_DB , which is the existing implementation and there is no change to it.  The DPU health information is stored in the switch ChassisStateDB.

Reply:
But the DPU runs "the same" SONiC as the host (where the common part works similarly), which stores the health information in the STATE_DB. The question is how this will be handled in the scope of this design. Will the information be duplicated in both DBs?

Reply:
The information will be duplicated in both DBs.